### PR TITLE
Fix: compile targeting Windows 10 Store

### PR DESCRIPTION
**Pull request by infosia**
[TIMOB-19677](https://jira.appcelerator.org/browse/TIMOB-19677) and [TIMOB-19678](https://jira.appcelerator.org/browse/TIMOB-19678)

Fix compile error when targeting Windows 10 Store App.

### CODE REVIEW

File: Examples/NG/src/Package.win10.appxmanifest.in [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="@PACKAGE_GUID@"
+    Publisher="CN=appcelerator"
+    Version="1.0.0.0" />
```
Comment (sgtcoolguy) [inline Examples/NG/src/Package.win10.appxmanifest.in]: Version="@VERSION@"

File: Examples/NG/src/Package.win10.appxmanifest.in [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="@PACKAGE_GUID@"
+    Publisher="CN=appcelerator"
```
Comment (sgtcoolguy) [inline Examples/NG/src/Package.win10.appxmanifest.in]: Publisher="@PUBLISHER_ID@"

File: Examples/NG/src/Package.win10.appxmanifest.in [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="@PACKAGE_GUID@" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
```
Comment (sgtcoolguy) [inline Examples/NG/src/Package.win10.appxmanifest.in]: PhoneProductId="@PHONE_PRODUCT_ID@" PhonePublisherId="@PHONE_PUBLISHER_ID@"

File: Examples/NG/src/Package.win10.appxmanifest.in [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+  <Properties>
+    <DisplayName>@SHORT_NAME@</DisplayName>
+    <PublisherDisplayName>Appcelerator</PublisherDisplayName>
```
Comment (sgtcoolguy) [inline Examples/NG/src/Package.win10.appxmanifest.in]: <PublisherDisplayName>@PUBLISHER_DISPLAY_NAME@</PublisherDisplayName>

Comment (infosia) [inline Examples/NG/src/Package.win10.appxmanifest.in]: Yes, they are supported when cmake is invoked via `ti build`, but `NG` examples doesn't have them.

**Pull request merged**